### ISSUE TITLE
EOS expired transaction errors

### ISSUE THREAD

**Issue by shinotenshi1977** [klesun/bcu-bugs-public#15]
I will start attaching all the recent conversation in bug chat regarding this one.

TL:DR = Regularly EOS will fail on first attempt of putting a transaction through and succeed on the second attempt.  The first attempt fails with an expired transaction error with expiry date time always set to Oct 13, 2021 7:22:32.000

Random potentially useful observations:
* The expiry date time is always the same and lines up with roughly when the errors started showing up.
* There doesn't appear to be a user end software issue as it's shown in all tested formats including: Wombat Android app, Wombat Chrome plugin, and using Private Key in Brave for Android.
* The Private Key confirmation screen in Brave browser has shown an account balance for myself (Shino Tenshi) that was consistent with the expiry date/time being shown.
* The display of outdated balance does not follow the error displaying.  It has shown both during immediately successful transactions and failed transactions.
* The correct balance has also shown during failed transactions.

Another issue appeared around the same time where some transactions don't react at all on the first push of the confirmation button.  The second push of the confirmation button produces a transaction compete popup with the cpu and net used, but doesn't change to the activity log like it normally does, it just remains on the confirmation page.  When this happens, the pending transaction doesn't show at all in the activity log, it only shows when the transaction is complete.  I'm not sure if this is related or a separate issue, but listing it here since it showed up around the same time and normally fails on the first attempt and succeeds on the second when it shows up.

**Comment by shinotenshi1977** [comment]
Confirmation screen showing correct balance but transaction failing anyway.

![Screenshot_20211017-181724](https://user-images.githubusercontent.com/92234799/137647813-0e7f8bbe-0af5-4212-9bb6-d6dcdc7d1342.png)
![Screenshot_20211017-181646](https://user-images.githubusercontent.com/92234799/137648527-a15b678b-851e-4458-9e5d-3575e4edf48a.png)

**Comment by shinotenshi1977** [comment]
Confirmation screen showing incorrect outdated balance.  This particular time the transaction went through anyway.

![Screenshot_20211017-175049](https://user-images.githubusercontent.com/92234799/137648587-697c748d-f15a-420f-af7f-1a93d6c6cdc4.png)

**Comment by klesun** [comment]
Hm, we had the `21.10.0` update on _Oct 13, 2021_... I it a coincidence, I wonder?

Upd.: hm, I don't see anything even closely related to eos transactions in this update code though...

**Comment by klesun** [comment]
Pedro reports that he gets same error from around 13th too, so it's at least two people.

**Comment by viracocha48** [comment]
It seems to have stopped doing that, maybe it was a transitory issue.

**Comment by viracocha48** [comment]
Think this should be safe to close as it hasn't been happening anymore. May have been some caching issue somewhere.
